service-loadbalancer/gce/README.md

```diff
@@ -0,0 +1,18 @@
+# GCE Load Balancers
+
+
+## Disclaimer:
+- This is a **work in progress**.
+- For fault tolerant load balancing of ingress traffic, you need:
+  1. Multiple hosts running load balancers
```

Comment:
heh, no i just copied that over from the other serviceloadbalancer README. I'm writing e2es atm, figured that would somehow validate this enough that you won't have to do some hardcore review.